### PR TITLE
feat: add media tab #9

### PR DESCRIPTION
**Pull request by timofei-maximov-saritasa**
#9

### CODE REVIEW

File: src/providers/settings.tsx [hunk 1]
```diff
@@ -22,6 +27,7 @@ const defaultSettings = {
     [TagClassification.Technical]: true,
   },
   spoilerLevel: SpoilerLevel.Major,
+  showNsfwImages: true,
```
Comment (prekel) [inline src/providers/settings.tsx]:
```suggestion
  showNsfwImages: false,
```

**Review by prekel — CHANGES_REQUESTED** [review]

File: src/features/VisualNovel/pages/MediaPage/MediaPage.module.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  display: flex; 
+  gap: 20px;
+  flex-wrap: wrap;
+}
+
+.image {
+  height: auto;
+  border-radius: 8px;
+  width: 320px;
+}
```
Comment (NightOfNights) [inline src/features/VisualNovel/pages/MediaPage/MediaPage.module.css]: Do you need `height: auto` property here? `img` and `video` tags have this by default

File: src/features/VisualNovel/pages/MediaPage/MediaPage.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  const filterPredicate = (screen: VisualNovelScreenshot): boolean => (settingsContext.showNsfwImages ? true : !screen.isNsfw);
+
+  if (isLoading) {
+    return <>Loading...</>;
+  }
+
+  if (error) {
+    return <>{`An error has occurred: ${error.message}`}</>;
+  }
+
```
Comment (NightOfNights) [inline src/features/VisualNovel/pages/MediaPage/MediaPage.tsx]: I think in the future we will need to replace this with our custom components for loading and errors. Maybe add TODO comment for this section?

File: src/providers/settings.tsx [hunk 4]
```diff
@@ -13,6 +13,11 @@ interface Settings {
    * Spoiler level for tags.
    */
   spoilerLevel: SpoilerLevel;
+
+  /**
+   * What images to show.
+   */
+  showNsfwImages: boolean;
```
Comment (NightOfNights) [inline src/providers/settings.tsx]: "What images to show" is strange comment for `boolean` variable, don't you think so 😄 ? Let's improve it

Comment (NightOfNights) [reply src/providers/settings.tsx]: Also rename this param to something like `isNsfwContentAllowed`

File: src/features/VisualNovel/pages/MediaPage/MediaPage.tsx (hunk 3 above)
Comment (timofei-maximov-saritasa) [reply src/features/VisualNovel/pages/MediaPage/MediaPage.tsx:27]: We are working on it right now.

Comment (Moltenship) [reply src/features/VisualNovel/pages/MediaPage/MediaPage.tsx]: We are working on it right now

**Review by NightOfNights — APPROVED** [review]

**Review by prekel — APPROVED** [review]

**Review by moetazaneta — APPROVED** [review]

**Pull request merged**